deploy/lib/createFunctions.js

```diff
@@ -183,6 +223,12 @@ module.exports = {
       domain_name: func.domain_name,
     };
 
+    // if set to true, ensure that we keep serverless config file as single source of truth
+    // (deleting other ressources not present in file)
+    if (func.single_source === null || func.single_source === undefined) {
+      // params.single_source = false;
```

Comment:
Same comment here, should this be commented out? If it's not, this block of code is a no-op.